docs/implementation.html

```diff
@@ -183,10 +187,15 @@
 <pre>
 On-disk format of a message
 
-message length : 4 bytes (value: 1+4+n)
-"magic" value  : 1 byte
+message length : 4 bytes (value: 4+1+1+8+4+K+4+V)
```

Comment:
perhaps worth reiterating here that the message length only is +8 if magic value is > 0 .